troposphere/static/js/components/common/InstanceDetail.jsx

```diff
@@ -59,7 +54,7 @@ const InstanceDetail = React.createClass({
     },
 
     renderActiveInstance: function(instance) {
-        let { InstanceStore, ProjectStore } = this.props.subscriptions;
+        let { ProjectStore } = this.props.subscriptions;
```

Comment:
Self note:
ImageStore is subscribed to in this file, without being used. Small change, more important take away, is that this is why you don't use strings to represent variables.